pkgs/kde/plasma/kinfocenter/default.nix

```diff
@@ -1,30 +1,64 @@
 {
+  aha,
+  clinfo,
+  dmidecode,
+  glxinfo,
+  iproute2,
   lib,
+  libusb1,
   mkKdeDerivation,
-  substituteAll,
+  pciutils,
+  pulseaudio,
   qttools,
-  xdpyinfo,
+  substituteAll,
   systemsettings,
-  libusb1,
+  util-linux,
+  vulkan-tools,
+  wayland-utils,
+  xdpyinfo,
 }:
+let
+  tools = {
+    aha = lib.getExe aha;
+    clinfo = lib.getExe clinfo;
+    dmidecode = lib.getExe' dmidecode "dmidecode";
+    eglinfo = lib.getExe' glxinfo "eglinfo";
```

Comment:
Should probably use mesa-demos

Reply:
That's very big and contains many more things we don't need.

Reply:
And glxinfo/eglinfo are horribly outdated.

Reply:
Should we alias the package then?

Reply:
#324562